client/weapons.lua

```diff
@@ -95,3 +92,12 @@ Citizen.CreateThread(function()
 		Citizen.Wait(0)
 	end
 end)
+
+CreateThread(function() -- Weapon Damage
+    for weapon, modifier in pairs(Config.DamageWeapons) do
+      local hash = GetHashKey(weapon)
+      local ped = PlayerPedId()
+      SetWeaponDamageModifier(hash, modifier)
+      SetPedSuffersCriticalHits(ped, false)
```

Comment:
I am not sure if this is needed. Maybe it should be a config flag?

Reply:
And it doesn't need to be in a for loop.